app/src/main/java/it/quip/android/fragment/CreateQuipComposeFragment.java

```diff
@@ -51,17 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_create_quip, container, false);
+        return inflater.inflate(R.layout.fragment_create_quip_compose, container, false);
     }
 
     @Override
     public void onViewCreated(View v, @Nullable Bundle savedInstanceState) {
-        mUser = getArguments().getParcelable(USER_KEY);
+        User user = QuipitApplication.getCurrentUser();
         mLlProfile = (LinearLayout) v.findViewById(R.id.quip_create_profile);
         mIvProfile = (ImageView) mLlProfile.findViewById(R.id.iv_profile);
         try {
             Picasso.with(getActivity())
-                    .load("drawable/edgarjuarez.jpeg")
+                    .load(user.getImageUrl())
```

Comment:
RIP...edgar rodriguo juarez...one of the best pitchers to ever take the mound for the tiajuana toros